pkg/apis/baremetal/v1alpha3/doc.go

```diff
@@ -1,4 +1,4 @@
-// +k8s:deepcopy-gen=package,register
+// +k8s:deepcopy-gen=package
```

Comment:
Interested to know what this change does.

Reply:
The `register` tag is deprecated, it was used to automatically register generated conversions with a default scheme (that is not used anymore). The tag is a no-op nowadays. See e.g. here: https://github.com/kubernetes/code-generator/commit/84812afcd0ac10d284abed95ae0d9e9f3a2d866a